crates/cairo-lint-core/tests/helpers/scarb.rs

```diff
@@ -0,0 +1,88 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+    sync::OnceLock,
+};
+
+use anyhow::{Context, Result};
+use cairo_lang_filesystem::db::CORELIB_CRATE_NAME;
+use indoc::indoc;
+use scarb_metadata::{Metadata, MetadataCommand};
+use tempfile::tempdir;
+use which::which;
+
+pub const SCARB_TOML: &str = "Scarb.toml";
+
+fn get_scarb_path() -> Result<PathBuf> {
+    which("scarb").map_err(|_| anyhow::anyhow!("`scarb` not found in `PATH`"))
+}
+
+/// Calls `scarb metadata` on an empty Scarb package to find the `core` package.
+fn get_scarb_metadata(manifest: &Path) -> Result<Metadata> {
+    let scarb_path = get_scarb_path()?;
+
+    MetadataCommand::new()
+        .scarb_path(scarb_path)
+        .manifest_path(manifest)
+        .inherit_stderr()
+        .exec()
+        .context("failed to execute: scarb metadata")
+}
+
+/// Try to find a Scarb-managed `core` package if we have Scarb toolchain.
+///
+/// The easiest way to do this is to create an empty Scarb package and run `scarb metadata` on it.
```

Comment:
I know we do it all across the board, but does maciej know that it's a widespread problem? Perhaps it'd be good to at least add it to the backlog so that sometime we can consider doing it the right way

Reply:
(also the case in LS)

Reply:
Will let Maciej know about this problem in Ls and Lint